doc/development-context-aware-applications/how-to-connect-to-external-context-providers.md

```diff
@@ -24,26 +24,20 @@ advantages:
 
 Let’s illustrate with an example based on the NiceEating case. Let’s
 consider an external restaurant booking system that is able to provide
-in real time the number of free places of a given restaurant and
-occupancy level (as a percentage). That system runs at
+in real time
```

Comment:
[...] in real time the occupancy level (as an absolute value) of a given restaurant.